sw/device/silicon_creator/mask_rom/BUILD

```diff
@@ -264,17 +267,11 @@ opentitan_functest(
     srcs = [
         "mask_rom_test.c",
     ],
-    targets = ["verilator"],  # Can only run in verilator right now.
+    signed = True,
     verilator = verilator_params(
-        args = [
-            "console",
-            # This test program is not signed, so a boot fault is the expected behavior.
-            "--exit-success='boot_fault: 0x0142500d'",
-        ],
-        rom = ":mask_rom_verilator_scr",
+        rom = ":mask_rom_sim_verilator_scr_vmem",
         tags = [
             "cpu:4",
-            "failing_verilator",
```

Comment:
Ah good point, one goal of this PR was to make it so un-signed flash test images would not be the barrier to this test still failing, which is fixed now, but there is still the UART issue we discussed earlier why the test fails, so I added "failing_verilator" tag back, until this can be fixed.